swap/src/protocol/bob/swap.rs

```diff
@@ -175,10 +177,14 @@ async fn next_state(
                 // Bob sends Alice his key
 
                 select! {
-                    _ = event_loop_handle.send_encrypted_signature(state.tx_redeem_encsig()) => {
+                    result = event_loop_handle.send_encrypted_signature(state.tx_redeem_encsig()) => {
+                        // TODO: Evaluate if that should bail?
```

Comment:
What is the state of this TODO?

Reply:
I am not sure how we should react to this one - in the long run we should definitely have some retry mechanism here. For now we can `bail` - I marked this one because it is different than the other cases where we waited for timelocks to expire.

Reply:
@rishflab Can we remove this hunk from this commit and just fix the handling of `send_encrypted_signature` entirely in the 3rd patch?